drift/lib/src/sentry_transaction_executor.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:drift/backends.dart';
+import 'package:meta/meta.dart';
+import 'package:sentry/sentry.dart';
+import 'sentry_span_helper.dart';
+
+/// @nodoc
+@internal
+class SentryTransactionExecutor extends TransactionExecutor {
+  final TransactionExecutor _executor;
+
+  final Hub _hub;
+
+  final _spanHelper = SentrySpanHelper(
+    // ignore: invalid_use_of_internal_member
+    SentryTraceOrigins.autoDbDriftTransactionExecutor,
+  );
+
+  final String? _dbName;
+
+  /// @nodoc
+  SentryTransactionExecutor(this._executor, Hub hub, {@internal String? dbName})
+      : _hub = hub,
+        _dbName = dbName {
+    _spanHelper.setHub(_hub);
+    beginTransaction();
```

Comment:
Calling the constructor is supposed to call `beginTransaction`? Isn't this behaviour we are changing?